### ISSUE TITLE
Forms are displayed under the sidebar

### ISSUE THREAD

**Issue by ghost** [phit/psychokiller-ts3web-bootstrap-theme#1]
It seems that some forms are placed under the left side menu instead of right next to it.

![forms_bootstrap_theme](https://cloud.githubusercontent.com/assets/5872449/22364701/8f06c14a-e473-11e6-90b8-6d60e283b8ef.jpg)

**Comment by phit** [comment]
yup, theme isn't done and I haven't had time to finish it so far
